osu.Game.Tests/Visual/UserInterface/TestSceneSettingsToolboxGroup.cs

```diff
@@ -1,19 +1,23 @@
-﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System.Linq;
 using NUnit.Framework;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
+using osu.Framework.Testing;
 using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays;
 using osu.Game.Screens.Play.HUD;
 using osu.Game.Screens.Play.PlayerSettings;
+using osuTK.Input;
 
-namespace osu.Game.Tests.Visual.Gameplay
+namespace osu.Game.Tests.Visual.UserInterface
 {
     [TestFixture]
-    public class TestSceneReplaySettingsOverlay : OsuTestScene
+    public class TestSceneSettingsToolboxGroup : OsuManualInputManagerTestScene
```

Comment:
I find this rename kind of weird. A `SettingsToolboxGroup` is not directly instantiated anywhere in this test. With the new test name, it looks like the test is testing something entirely different than you may think it's testing from the name alone when you read what it's actually doing.

Reply:
I don't really even think the rename was that much needed in the first place.

Reply:
The original name had me thinking there was no test coverage for `SettingsToolboxGroup` multiple times before.

Reply:
Have removed any references to "Player" and instantiated groups using `SettingsToolboxGroup` as you mentioned.